Modules/_sqlite/row.c

```diff
@@ -187,19 +187,22 @@ static Py_hash_t pysqlite_row_hash(pysqlite_Row *self)
     return PyObject_Hash(self->description) ^ PyObject_Hash(self->data);
 }
 
-static PyObject* pysqlite_row_richcompare(pysqlite_Row *self, PyObject *_other, int opid)
+static PyObject *
+pysqlite_row_richcompare(pysqlite_Row *self, PyObject *_other, int opid)
```

Comment:
Nit: I prefer to not include cosmetic-only changes in such patch. Unfortunately, this isn't only case where sqlite3 codebase doesn't follow our general conventions, but I'd like to change them only when we need to change the definition `pysqlite_row_richcompare()`.